crates/autopilot/src/infra/persistence/dto/order.rs

```diff
@@ -35,10 +35,12 @@ pub struct Order {
     pub post_interactions: Vec<boundary::InteractionData>,
     pub sell_token_balance: boundary::SellTokenSource,
     pub buy_token_balance: boundary::BuyTokenDestination,
+    #[serde(flatten)]
```

Comment:
Is this change related?

Reply:
Yes, [I erased the `Order` type from `driver_model`](https://github.com/cowprotocol/services/pull/2250/files#diff-53c95c46cb3be0831e6245ae68a96c00e93973b8308f6036f4562de00b3b27c1L127-L131) and reused this `persistence::dto::Order` which is using `OrderClass` which needs to be flattened to make it equal.